### DISCUSSION TITLE
Risk-First Diagrams Explained

### DISCUSSION THREAD

**Discussion by robmoffat** [risk-first/website#76] [Team Posts]
I met up with @kooba on Monday, which led to an interesting discussion about Risk-First, and also prompted me to write  https://riskfirst.org/Risk-First-Diagrams, which breaks down what is going on in the diagrams on the site.

Also, in response to @andreadellacorte's suggestions below, I've tried to improve the navigation of the site somewhat with numbered beads down the left-hand-side to indicate where you are in a sequence of articles.  

Hopefully these are moving things in the right direction!  

Lots more articles in the works at the moment, after the summer lull.

thanks,
Rob

**Comment by iliaskarim** [comment]
The article looks great @robmoffat. I noticed a typo in "unweildy" which should be spelled "unwieldy." I found your diagram explanations helpful.
